reference/5.1/Microsoft.PowerShell.Core/About/about_Object_Creation.md

```diff
@@ -135,12 +135,56 @@ You can use the following sample to determine what .NET types are currently
 loaded for you to instantiate.
 
 ```powershell
-[AppDomain]::CurrentDomain.GetAssemblies() | ForEach-Object {
-                $_.GetExportedTypes() | ForEach-Object { $_.FullName }
-} | Out-Host -Paging
+[AppDomain]::CurrentDomain.GetAssemblies() |
+  ForEach-Object {
+    $_.GetExportedTypes() |
+      ForEach-Object { $_.FullName }
+  }
 ```
 
-### System.Activator class
+Objects created using the **new()** method may not have the same properties as
+objects of the same time that are created by PowerShell cmdlets. PowerShell
```

Comment:
Should this be `objects of the same type`?

Reply:
oops